updates/version.yaml

```diff
@@ -23,3 +23,4 @@
 1.0.23: Added support for balloon-selector field type, added Brazilian Portuguese translation, fixed some bugs
 1.0.24: Added support for tag list field type, added read only toggle for fields. Prevent plugins from using reserved PHP keywords for class names and namespaces
 1.0.25: Allow editing of migration code in the "Migration" popup when saving changes in the database editor.
+1.0.26: Added migration editor button to add an 'id' column to a table.
```

Comment:
@DanHarrin I will combine this update with the other updates waiting for the next version, so could you remove the `updates/version.yaml` change for us?